source/user/mcscf.rst

```diff
@@ -1,57 +1,299 @@
-.. _theory_mcscf:
+.. _user_mcscf:
 
 Multi-configuration self-consistent field (MCSCF)
 *************************************************
 
 *Modules*: :mod:`mcscf`
 
 
+Introduction
+------------
 
-Symmetry in CASSCF
-------------------
+Multiconfigurational self-consistent field (MCSCF) methods extend Hartree-Fock (HF) theory by describing the wave function as a linear combination of determinants. 
+Unlike full configuration interaction (FCI), described in :numref:`theory_ci`, complete active space (CAS) methods perform the FCI procedure on a subset of the molecular orbitals, referred to as the "active space." 
+These methods are crucial for systems that exhibit strong electron correlation, such as transition metal complexes.
+For a detailed discussion of MCSCF methods, we direct the reader to :cite:`Helgaker2013` and :cite:`esqc`.
 
-.. literalinclude:: /../examples/mcscf/21-nosymhf_then_symcasscf.py
+The MCSCF module has two main flavors: CASCI and CASSCF. 
+CASCI uses the single reference orbitals from a HF calculation and solves the CI eigenvalue problem once.
+CASSCF calculations optimize the orbitals for the set of determinants and require solving the CI eigenvalue multiple (and possibly many) times.
 
-To restart a CASSCF calculation, you need prepare either CASSCF orbitals
-or CI coefficients (not that useful unless doing a DMRG-CASSCF calculation) or
-both.  For example:
+Minimal examples for each type of calculations are shown below.
 
-.. literalinclude:: /../examples/mcscf/13-restart.py
+CASCI
+"""""
+.. code-block:: python
 
+  import pyscf
+  mol = pyscf.M(
+      atom = 'O 0 0 0; O 0 0 1.2',
+      basis = 'ccpvdz',
+      spin = 2)
+  myhf = mol.RHF().run()
+  ncas, nelecas = (6,8)
+  mycas = myhf.CASCI(ncas, nelecas).run()
```

Comment:
Isn't this a bit of a funny example? How accurate are the HF orbitals?

Reply:
This is just the trimmed down version of this example:

https://github.com/pyscf/pyscf/blob/7be5e015b2b40181755c71d888449db936604660/examples/mcscf/00-simple_casci.py#L10-L20

Do you mean accurate in the sense that we'll get meaningful CASCI results?

Reply:
Yes. IIRC HF virtuals usually give pretty poor estimates, which is why people have come up with a bunch of methods to get better virtual orbitals to match the HF occupied orbitals.

Reply:
Ok, that's a fair point, but I think the goal of these examples is to give people a minimum working example and favor ease of understanding rather than strict correctness (in terms of the results). That said, if people are strongly opposed to this, I'm happy to adjust the example.

Reply:
I mean.. it would make a lot more sense if did the CAS-CI calculation in a basis of CISD natural orbitals :upside_down_face: